.pipelines/shared-cluster.yml

```diff
@@ -0,0 +1,93 @@
+# No PR triggers as it runs on a schedule
+pr: none
+trigger: none
+
+schedules:
+- cron: "0 1 * * Sun"
+  displayName: ⭕ Rebuild shared cluster Sun 1am (UTC)
+  always: true
+  branches:
+    include:
+    - master
+
+variables:
+- template: vars.yml
+  LOCATION: eastus
+  RESOURCEGROUP: v4-eastus
+  CLUSTER: aro-shared
+  VNET: dev-vnet
+
+jobs:
+- job: ⏬ Tear down shared cluster
+  pool:
+    vmImage: ARO-RP
```

Comment:
There does this comes from? 
https://msazure.visualstudio.com/AzureRedHatOpenShift/_settings/agentqueues